scripts/clear-all-fault-leds.sh

```diff
@@ -4,6 +4,14 @@
 # /xyz/openbmc_project/State/Decorator/OperationalStatusManager
 # to true or false.
 
+# Skip running this script during a warm reboot(graceful/force).
```

Comment:
This is running even before the argument/usage checks. Please run this after those basic checks are done.

Reply:
done